modules/otmBidAdapter.js

```diff
@@ -33,43 +46,37 @@ export const spec = {
     logInfo('validBidRequests', validBidRequests);
 
     const bidRequests = [];
-    let tz = new Date().getTimezoneOffset()
-    let referrer = '';
-    if (bidderRequest && bidderRequest.refererInfo) {
-      referrer = bidderRequest.refererInfo.referer;
-    }
+    const tz = new Date().getTimezoneOffset()
+    const referrer = bidderRequest && bidderRequest.refererInfo ? bidderRequest.refererInfo.referer : '';
 
     _each(validBidRequests, (bid) => {
-      let domain = getValue(bid.params, 'domain') || ''
-      let tid = getValue(bid.params, 'tid')
-      let cur = getValue(bid.params, 'currency') || DEF_CUR
-      let bidid = getBidIdParameter('bidId', bid)
-      let transactionid = getBidIdParameter('transactionId', bid)
-      let auctionid = getBidIdParameter('auctionId', bid)
-      let bidfloor = _getBidFloor(bid)
+      const domain = isStr(bid.params.domain) ? bid.params.domain : getTopDomain()
```

Comment:
prebid already gets this for you with bidderRequest.refererInfo, which you grab above. Why are you adding this function?

Reply:
in other words, https://github.com/prebid/Prebid.js/blob/master/src/refererDetection.js does this work for you, the rule to use our methods is to keep build sizes trim. If every adapter reproduced, build size would grow uncontrollably.

Reply:
now I see what u mean earlier, thx!
changed to use URL from `bidderRequest.refererInfo.referer`